src/components/__tests__/Logs.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Logs from '../logs/Logs';
+import renderer from 'react-test-renderer';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+
+it('renders correctly', () => {
+  const exampleLog = `\
+hello world
+I am a log
+I log things
+`;
+
+  const tree = renderer
+    .create(
+      <Router>
+        <Switch>
+          <Route exact path="/">
+            <Logs logs={exampleLog} />
```

Comment:
Can we just ignore Router/Switch/Route elements and just test logs?

Reply:
Because it has navigation functionality in the component it requires this to work.